com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ModelServerRouting.java

```diff
@@ -107,6 +115,16 @@ public void bindRoutes() {
 		
 		return Optional.empty();
 	}
+
+	private Optional<String> getModelUriQueryParam(Context ctx) {
+		if (ctx.queryParamMap().containsKey("modeluri"))
+			return Optional.of(ctx.queryParamMap().get("modeluri").get(0))
+				.map(modelUri -> modelUri.replace("file://", ""))
+				.map(modelUri -> modelUri.replace(serverConfiguration.getWorkspaceRoot(), ""))
```

Comment:
Will this work if the user's workspace path has spaces in it and the URI has properly escaped them?  And should the URI only be relativized if the parameter value is a `file:` scheme URI?  Perhaps we should use the `URI` API to relativize the parameter if it is an absolute file URI.

Reply:
I've created follow-up issue #69 to track this.